src/pylon_camera/pylon_camera_parameter.cpp

```diff
@@ -204,6 +207,16 @@ void PylonCameraParameter::readFromRosParameterServer(const ros::NodeHandle& nh)
             }
         }
     }
+
+
+    
+    if ( nh.hasParam("whitebalance_auto") )
+    {
+        nh.getParam("whitebalance_auto", whitebalance_auto_);
+        std::cout << "continuous auto white balance set" << std::endl;
```

Comment:
ROS_INFO_STREAM is fine too - as long as the output goes through the ROS logging system.